File: Python/Palindrome.py
num = int(input("Enter a number"))
res=0 
temp=num

while num>0:
  i=num%10
  res=res*10+i
  num=num//10
if temp==res:
  print(temp,"is palindrome")
else:
  print(temp,"is not Palindrome")

File: Java/Palindrome.java
package Java;

import java.util.Scanner;
//Palindrome

class Palindrome {

  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    System.out.println("Entar a number: ");
    int num = sc.nextInt();

    int rev = 0;
    int temp = num;

    while (num > 0) {
      int sum = num % 10;
      rev = rev * 10 + sum;
      num = num / 10;

    }
    if (temp == rev) {
      System.out.println("Palindrome");

    } else {
      System.out.println("Not Palindrome");
    }
  }

}